Cargo.toml

```diff
@@ -62,12 +63,13 @@ rustdoc-args = ["--cfg", "docsrs"]
 name = "webpki"
 
 [features]
-default = ["std"]
+default = ["std", "ring"]
+ring = ["dep:ring"]
```

Comment:
is this `dep:xxx` syntax to disambiguate from a feature with the same name as the crate, or another purpose?

Reply:
So traditionally optional dependencies would create sort of an implicit feature of the same name. However, that's not very obvious. That's why we added this `dep:` syntax (I worked on this in like 2017-2018, but it took ages to stabilize), which both makes it clearer that the feature exists and allows you to put extra dependencies in (for example, if we only wanted to rely on untrusted in case ring is enabled).

Reply:
Ahhh! That makes sense, thank you. Also cool to hear you worked on it :-)